src/eu/packages/eu-component-link/link.story.js

```diff
@@ -0,0 +1,85 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/eu-resources-icons/dist/sprites/icons.svg';
+import uiIcons from '@ecl/eu-resources-icons/dist/lists/ui.json';
+
+import linkDocs from './docs/link.md';
+import link from './link.html.twig';
+
+const iconPositionSettings = {
+  before: 'before',
+  after: 'after',
+};
+
+const iconsList = {};
+iconsList.none = null;
+
+uiIcons.forEach(icon => {
+  iconsList[icon] = icon;
+});
+
+storiesOf('Components/Link', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withNotes)
+  .addDecorator(withCode)
+  .add(
+    'default',
+    () => {
+      const iconPosition = select(
+        'Icon position',
+        iconPositionSettings,
+        'after'
+      );
+
+      const iconsListSelect = select('Icon (sample)', iconsList, null);
+
+      return link({
+        link: {
+          type: 'default',
+          label: text('Label', 'Default link'),
+          path: '/example#link-default',
+          icon_position: iconPosition,
+        },
+        icon: {
+          type: 'ui',
+          name: iconsListSelect,
+          path: defaultSprite,
+        },
```

Comment:
we should set the icon size to `fluid` here, it is currently too big
the same for the other icon below

Reply:
Fixed in https://github.com/ec-europa/ecl-twig/pull/31/commits/45a6305895c369887c37648d7ec8c93b36c7d32b